<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Category</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbColor">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grey</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Orange</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Purple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pink</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Natural</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbProfile">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Proxy</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbCategory">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jackets</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sweatshirts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tops/Sweaters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T-Shirts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pants</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hats</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bags</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Accessories</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Skate</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string> Size</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="txtProxy"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Colour</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPlainTextEdit" name="txtKeyword">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Keyword, each keyword per line&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbSize">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Small</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Large</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cbStatus">
     <item>
      <property name="text">
       <string>Stop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Running</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Profile</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Status</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Kewords</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtKeyword</tabstop>
  <tabstop>cbSize</tabstop>
  <tabstop>cbCategory</tabstop>
  <tabstop>cbColor</tabstop>
  <tabstop>cbProfile</tabstop>
  <tabstop>txtProxy</tabstop>
  <tabstop>cbStatus</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
